waiter/config-full.edn

```diff
@@ -107,10 +107,16 @@
 
  ; ---------- Node ----------
 
+ ; The fully qualified domain name for this router
+ :router-fqdn "route1.waiter.example.com"
```

Comment:
Let's move this to `server-options` or `zookeeper`?

Reply:
I originally included these keys in `server-options`, but I moved them out because they aren't used by the Jetty server (which is the primary purpose of `server-options`), and I thought they confused things since the Jetty server won't actually respond to requests sent to that fqdn/port combination. We could include them in `zookeeper`, since they're currently only leveraged in service discovery. However, if we wanted to allow for them to be more generally used, then we could introduce some new sub-map with a key like `fqdn-config`?

Reply:
Maybe `router-config` ?